.github/workflows/ci-aarchxx-cross.yml

```diff
@@ -369,26 +273,18 @@ jobs:
         CI_TRIGGER: ${{ github.event_name }}
         CI_BRANCH: ${{ github.ref }}
 
-    - name: Send failure mail to dynamorio-devs
-      if: failure() && github.ref == 'refs/heads/master'
-      uses: dawidd6/action-send-mail@v2
+  send-failure-notification:
+      uses: ./.github/workflows/failure-notification.yml
+      needs: [aarch64-cross-compile, arm-cross-compile, android-arm-cross-compile, android-aarch64-cross-compile, a64-on-x86]
+      # We need this to ensure this job runs even if there is failure in
+      # any of the workflows marked as needed.
```

Comment:
Is there a wording error here?  "even if there is failure in any": ?  Do you mean "no failure"?  I don't understand this sentence.  Applies to each file.

Reply:
See https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds. Default behavior is that a failure in a needed job causes the dependent job to not run.

Reply:
> Is there a wording error here?

So no, no wording error here. If you have a suggestion to improve clarity I can apply it here, because I don't see how.

Reply:
I think I had in my head that there was a trigger for failure that was running the notification only on failure, so this sentence seemed backward.  It makes more sense now.  Maybe add at the end " (by default dependent jobs are not run on main job failure)" to really make it clear.

Reply:
Added something to that effect.